src/main/java/nl/tudelft/jpacman/npc/NPC.java

```diff
@@ -3,19 +3,29 @@
 import nl.tudelft.jpacman.board.Direction;
 import nl.tudelft.jpacman.board.Unit;
 
+
 /**
  * A non-player unit.
  * 
  * @author Jeroen Roosen 
  */
 public abstract class NPC extends Unit {
 
+	protected NPC() {
+	}
+
 	/**
 	 * The time that should be taken between moves.
 	 * 
 	 * @return The suggested delay between moves in milliseconds.
 	 */
 	public abstract long getInterval();
+	
+	protected String strategy;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/mathieu-craeye/jpacman-framework_2/file/2532197009/issues/source?bid=3267846&fileBranchId=3284210#l24)